CONFIGURATION.md

```diff
@@ -149,13 +149,22 @@ parameter as well. See further below for information on that.
 # right above the "start a new recording" button. This button links to the
 # value of this key. Setting this value is usually done via GET parameters, as
 # a global configuration often does not make sense. You also probably want to
-# set `return.label`.
+# set `return.label`. Also see `return.allowedDomains`.
 #target = "https://my-lsm.me.com/course?id=123"
 
 # Label for the site that `return.target` links to. Usually the name of your
 # LMS. The English button label is "Exit and go back to {{return.label}}". This
 # value is only used if `return.target` is set!
 #label = "Stud.IP"
+
+# A list of domains that are allowed for `return.target` links. The domain on
+# which Studio is served is always allowed. This is for security, as
+# redirecting to URLs that could be set by an attacker is a bad idea. See here:
+# https://cheatsheetseries.owasp.org/cheatsheets/Unvalidated_Redirects_and_Forwards_Cheat_Sheet.html
```

Comment:
The second part is not really relevant for someone configuring this and long tests just lead to people not reading this.

```suggestion
# which Studio is served is always allowed.
```